### ISSUE TITLE
Reaper fails to return Endpoint state

### ISSUE THREAD

**Issue by FlorianHockmann** [thelastpickle/cassandra-reaper#186]
The web interface as well as the REST API via `/cluster/{cluster_name}` don't return any information about the state of my Cassandra nodes. However, the correct information is present in the logs of Reaper:
```
DEBUG  [2017-09-05 14:11:32,704] [dw-32 - GET /cluster/titantestcassandracluster] c.s.r.r.ClusterResource - All Endpoints State /10.10.11.106
  generation:1504260044
  heartbeat:1072294
  SEVERITY:0.0
  HOST_ID:39d4f098-c80e-41ee-a2e1-70f03fb727fe
  RELEASE_VERSION:2.1.9
  RPC_ADDRESS:10.10.11.106
  NET_VERSION:8
  STATUS:NORMAL,-1033825457445431942
  LOAD:2.037125748E9
  SCHEMA:5171f5a9-e309-3a68-a497-563b62a44e23
  RACK:rack1
  DC:datacenter1
/10.10.11.107
  generation:1504260116
  heartbeat:1072068
  SEVERITY:0.0
  HOST_ID:4f53b336-9566-4fbe-a0b6-cf14ac75d3b5
  RELEASE_VERSION:2.1.9
  RPC_ADDRESS:10.10.11.107
  NET_VERSION:8
  STATUS:NORMAL,-1009285190475330802
  LOAD:2.04557441E9
  SCHEMA:5171f5a9-e309-3a68-a497-563b62a44e23
  RACK:rack1
  DC:datacenter1
/10.10.11.108
  generation:1504260186
  heartbeat:1071850
  SEVERITY:0.0
  HOST_ID:d9514ff0-a32f-40f3-a123-e86d5d0137c6
  RELEASE_VERSION:2.1.9
  RPC_ADDRESS:10.10.11.108
  NET_VERSION:8
  STATUS:NORMAL,-1052236183763570721
  LOAD:2.036328901E9
  SCHEMA:5171f5a9-e309-3a68-a497-563b62a44e23
  RACK:rack1
  DC:datacenter1
```

I'm using **Reaper 0.6.2** and **Cassandra 2.1.9**. 

Here is the `node_status` element from the response to `/cluster/{cluster_name}` (I just included the data for one host):
```js
"nodes_status": {
        "endpointStates": [
            {
                "sourceNode": "10.10.11.106",
                "endpoints": {
                    "Not available": {
                        "Not available": [
                            {
                                "endpoint": "10.10.11.106",
                                "hostId": "Not available",
                                "dc": "Not available",
                                "rack": "Not available",
                                "status": "Not available",
                                "severity": 0,
                                "releaseVersion": "Not available",
                                "tokens": "Not available",
                                "load": 0
                            }
                        ]
                    }
                },
                "totalLoad": 0
            }
        ]
    }
```
That's how it looks in the web interface:
![cassandra-reaper-webui-nodestatus_unavailable](https://user-images.githubusercontent.com/21224336/30061837-411acdf8-9249-11e7-8de0-bd9fbbbbd26d.jpg)

The output from running `bin/spreaper status-cluster {cluster_name}`:
[spreaper_status-cluster.txt](https://github.com/thelastpickle/cassandra-reaper/files/1277441/spreaper_status-cluster.txt)
The output from `nodetool gossipinfo`:
[nodetool_gossipinfo.txt](https://github.com/thelastpickle/cassandra-reaper/files/1277440/nodetool_gossipinfo.txt)

**Comment by adejanovski** [comment]
Fixed by #187 

Let's close this when the PR gets merged.

**Comment by adejanovski** [comment]
Fix has been merged.
